### PR TITLE
Add Odyssey blog post

### PR DESCRIPTION
**Pull request by ehazlett**

### CODE REVIEW

File: content/en/blog/ksynth-k8s-operator.md [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+      method: GET
+      period: 10s
+      expiry: 5s
+```
+
+That's it. When the controller detects this resource is created, it will configure the agent to perform a fetch to the
+service `demo` on port 8080 every 10 seconds. Odyssey automatically resolves the `demo` service endpoint making the
+check work on any environment.
+
+Once we have defined our checks we can view them in the [Kentik Portal](https://www.kentik.com). This gives teams
```
Comment (i3149) [inline content/en/blog/ksynth-k8s-operator.md:127]: Worth it to go into more detail on how to define these checks in the portal? That's the part I would worry gets hard.

File: content/en/blog/ksynth-k8s-operator.md [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+increasing load times between application versions? Is there a spike in traffic at certain hours
+that trigger slower responses? Are there other processes that are running that are negatively
+affecting the service but not to the point where it falls over? As Kubernetes use increases,
+neighboring applications can cause significant issues if not configured properly.
+When synthetics are used in combination with your monitoring and observability stacks, they provide
+a powerful view into the overall health of the application and help identify issues
+before they cause an outage.
+
+## Synthetics
+
```
Comment (i3149) [inline content/en/blog/ksynth-k8s-operator.md:23]: I feel like these two paragraphs kinda say the same thing? Abstract and Introduction to a paper.

File: content/en/blog/ksynth-k8s-operator.md [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+Ensuring your application is available, healthy, and performant is vital for any
+production service. Monitoring and observability critical component to notify when an application
+has an issue but what about the data leading up to that threshold? Synthetics give
+you continuous data points on the factors that matter most to your application whether
+it be a simple TCP ping, an HTTP status, or network trace. Does your application have
+increasing load times between application versions? Is there a spike in traffic at certain hours
+that trigger slower responses? Are there other processes that are running that are negatively
+affecting the service but not to the point where it falls over? As Kubernetes use increases,
+neighboring applications can cause significant issues if not configured properly.
+When synthetics are used in combination with your monitoring and observability stacks, they provide
```
Comment (i3149) [inline content/en/blog/ksynth-k8s-operator.md:18]: Aren't synthetics a part of monitoring/o11y stacks?

**Review by i3149 — APPROVED** [review]
Added a few thoughts.

File: content/en/blog/ksynth-k8s-operator.md (hunk 3 above)
Comment (ehazlett) [reply content/en/blog/ksynth-k8s-operator.md:18]: Ya I was trying to speak to the applications specifically.  I'll clarify :+1:

File: content/en/blog/ksynth-k8s-operator.md (hunk 1 above)
Comment (ehazlett) [reply content/en/blog/ksynth-k8s-operator.md:127]: These checks are all automated from the Odyssey controller.  There are no user defined checks in the portal for Odyssey - just via the spec.

Comment (ehazlett) [reply content/en/blog/ksynth-k8s-operator.md:127]: I will mention the other types though +1

**Review by i3149 — APPROVED** [review]
:rocket: :rocket:

**Pull request merged**